.github/workflows/ci.yaml

```diff
@@ -132,6 +132,11 @@ jobs:
           LITESTAR_DOCS_IGNORE_MISSING_EXAMPLE_OUTPUT: 1
         run: poetry run make docs
 
+      - name: Check docs links
```

Comment:
I think instead of having this as an explicit step, we should just include it in the build step. That's easier and takes less time

Reply:
Ah you mean moving it to the 'build docs' step like this?

```yaml
      - name: Build docs
        env:
          LITESTAR_DOCS_IGNORE_MISSING_EXAMPLE_OUTPUT: 1
        run: |
          poetry run make docs
          poetry run make docs-linkcheck
```

Reply:
Ah, never mind. I thought I remembered there being an option that would allow you to linkcheck while building, which I would have preferred, since now the docs building takes longer than the actual tests which isn't great, but it seems like such an option does not exist so :shrug: